src/commands/new.ts

```diff
@@ -7,13 +7,15 @@ import { resolve } from 'path';
 
 const { writeFile, readFile } = fPromises;
 const DEFAULT_ASYNCAPI_FILE_NAME = 'asyncapi.yaml';
+const DEFAULT_ASYNCAPI_TEMPLATE = 'basic.yaml';
```

Comment:
Did you mean `simple.yaml`?

```suggestion
const DEFAULT_ASYNCAPI_TEMPLATE = 'simple.yaml';
```

Reply:
Thanks @fmvilas good spot!